mlflow/entities/trace_metrics.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import dataclass
+from enum import Enum
+
+from mlflow.entities._mlflow_object import _MlflowObject
+from mlflow.protos import service_pb2 as pb
+
+
+class MetricsViewType(str, Enum):
+    TRACES = "TRACES"
```

Comment:
what does `str` of this enum output look like?

Reply:
It looks like `'MetricsViewType.TRACES'`

Reply:
let's fix it to `TRACES` for convenience.